src/index.ts

```diff
@@ -8,13 +8,14 @@
  * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
  */
 'use strict';
-const dom5 = require('dom5');
-const parse5 = require('parse5');
-const shadyCSS = require('shady-css-parser');
-const stream = require('stream');
+import File = require('vinyl');
+import * as dom5 from 'dom5';
+import * as parse5 from 'parse5';
+import {Transform} from 'stream';
+import {Stringifier as CSSStringifier, Parser as CSSParser} from 'shady-css-parser';
 
-class NoCommentStringifier extends shadyCSS.Stringifier {
-  comment(node) {
+class NoCommentStringifier extends CSSStringifier {
+  comment(node: {value: string}): string {
```

Comment:
Typescript is telling me it has implicit type any. But I had assumed that you have to define the type when extending a class function, since you could be defining a different signature for the function. Have you seen otherwise?

Eitherway, updating to use the more accurate `Comment` type from shady-css